integration/runfile-helpers.js

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
Could we call this file something more specific? I generally like to avoid anything like `helpers` or `utils`. I would be fine being super verbose/explicit with something like `npm-packages-from-bazel-runfiles.ts`

Reply:
👍